src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ sender, body, timeStamp, id, isLiked, onLikeMessage }) => {
+  const chatStyle =
+    sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
+
+  const onLikeClick = () => {
+    onLikeMessage({
```

Comment:
From my above comment in App.js, if you rename onLikeMessage to updateChatEntry then this method would be even more readable.